include/seqan3/alphabet/quality/all.hpp

```diff
@@ -35,9 +35,9 @@
 #pragma once
 
 #include <seqan3/alphabet/quality/aliases.hpp>
-#include <seqan3/alphabet/quality/quality_composition.hpp>
+#include <seqan3/alphabet/quality/qualified.hpp>
 #include <seqan3/alphabet/quality/concept.hpp>
-#include <seqan3/alphabet/quality/illumina18.hpp>
+#include <seqan3/alphabet/quality/phred42.hpp>
 
```

Comment:
Please add the `quality` submodule definition in this file, similar to the `nucleotide` submodule definition. You can then make all your phred types `\ingroup quality`.

Reply:
This would also be a good place to give an overview over the module (which data types are there, why did we chose to make these ones). I would also include the overview from:
https://en.wikipedia.org/wiki/FASTQ_format#Encoding

But the whole description thing can also be done at a later point in time; its not crucial for this PR.